### DISCUSSION TITLE
Unable to connect to AWS Elasticache for Redis AUTH CLUSTER

### DISCUSSION THREAD

**Discussion by thuylevn** [apache/superset#25767] [General]
Unable to connect to AWS Elasticache for Redis AUTH CLUSTER

{
        'CACHE_TYPE': 'RedisClusterCache',
        'CACHE_DEFAULT_TIMEOUT': 60 * 60 * 24, # 1 day default (in secs)
        'CACHE_KEY_PREFIX': 'superset_results',
        'CACHE_REDIS_CLUSTER': 'cluster_endpoint:6379',
        'CACHE_REDIS_PASSWORD': 'cluster_password',
    }

**Comment by thuylevn** [comment]
Python-flask version 2.0.1
redis version 4.2.0
Flask cache version 2.0.2

**Reply by thuylevn** [reply]
any helps?

**Reply by AnhQKatalon** [reply]
Hello @thuylevn ,

Did you figure out how to configure Superset to run with Redis Cluster mode? I also intend to use AWS ElastiCache for Superset but still have not found the correct way to set it up. If you already find a way, please let me know.

Thank you so much
